lending/loan_management/doctype/loan/test_loan.py

```diff
@@ -278,6 +278,7 @@ def test_loan_with_security(self):
 		self.assertEqual(loan.loan_amount, 1000000)
 
 	def test_loan_disbursement(self):
+		frappe.db.set_value("Company", "_Test Company", "enable_loan_accounting", 1)
```

Comment:
No need to do this for every test
1. Make this a decorator (check for similar settings in other files)
2. Enable accounting in the setup since all other tests assume accounting is enabled
3. In the specific tests, where we are testing accounting is disabled, disable the setting